MatrixSDK/VoIP/MXCallManager.m

```diff
@@ -403,7 +464,14 @@ - (void)handleCallInvite:(MXEvent *)event
         MXCall *call = [self callWithCallId:content.callId];
         if (!call)
         {
-            call = [[MXCall alloc] initWithRoomId:event.roomId andCallManager:self];
+            //  TODO: Check below logic is correct (and necessary)
```

Comment:
as long as mxcall expects the room it's suypposed to appear in, seems good (in the js-sdk the call object takes the signalling room)